src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -367,6 +453,20 @@ class CreatePipelineRun extends React.Component {
               />
             </FormGroup>
           )}
+          <FormGroup legendText="">
```

Comment:
should the legend text be applied to the FormGroup instead or can we remove the group here?

Reply:
The current style has the legend text inline with the Add & Remove buttons. We can't do this with a normal FormGroup legendText. So, instead, I'm using the styles that were in the SecretModal which make it look like a FormGroup's legendText.